tools/cli/commands/build.js

```diff
@@ -831,10 +857,33 @@ async function buildProject( t ) {
 		await fs.writeFile( `${ buildDir }/.gitattributes`, rules, { encoding: 'utf8' } );
 	}
 
+	// Get the proeject version number from the changelog.md file.
+	let projectVersionNumber = '';
+	try {
+		const changelogText = await fs.readFile( `${ t.cwd }/CHANGELOG.md`, { encoding: 'utf8' } );
+		projectVersionNumber = changelogText
+			.toString()
+			.split( '\n' )
+			.find( line => {
+				return line.startsWith( '## ' ) && ! line.includes( '-alpha' );
+			} )
+			.split( ' ' )[ 1 ]
+			.replace( /[[\]]/g, '' );
```

Comment:
You might use a regex to match the line and extract the version number instead. See for example [this](https://github.com/Automattic/jetpack/blob/2d43693d034f4a3a964579852e6b3f39130d7a3e/.github/files/gh-autotagger/workflows/autotagger.yml#L53) and [this](https://github.com/Automattic/jetpack/blob/2d43693d034f4a3a964579852e6b3f39130d7a3e/projects/packages/changelogger/lib/KeepAChangelogParser.php#L172).

Reply:
Thanks for the example! 42eb636a1ed020255cc7809467531984b60b246e